static/app/views/performance/transactionSummary/transactionAnomalies/anomalyChart.tsx

```diff
@@ -54,7 +55,8 @@ const _AnomalyChart = (props: Props) => {
     yAxis: {
       axisLabel: {
         // Coerces the axis to be count based
-        formatter: (value: number) => axisLabelFormatter(value, 'tpm()'),
+        formatter: (value: number) =>
+          axisLabelFormatter(value, aggregateOutputType('tpm()')),
```

Comment:
In the spirit of the Jira ticket, this should probably be
```suggestion
        formatter: (value: number) =>
          axisLabelFormatter(value, 'duration'),
```

right? Then you can remove the comment above that says "Coerces the axis etc etc"

Reply:
I believe the right unit for `tpm()` should be either `number` or even `integer`

Reply:
Just double checked, should be 'number'! Made the change